### ISSUE TITLE
Provide a way to list pending issues/suggestions/translations

### ISSUE THREAD

**Issue by anter69** [codewars/codewars.com#1431]
Currently when I complete a kata, I check for pending issues/suggestions/translations, and try to resolve them. But it would be more efficient if these could be listed somehow. Thus, please provide a way to list pending issues/suggestions/translations, 

Also, I see many pending translations from power users that are simply "forgotten". Although I don't mean to approve them blindly, but those translations are usually good enough. Unfortunately `/users/username/published_translations` doesn't show anything... (see #1432)

**Comment by kazk** [comment]
Is this about the kata discourse page? If so, I think the current
```
All | Issues (9) | Questions (2) | Suggestions (6)
```
can be changed to
```
All | Issues (3) | Questions (1) | Suggestions (1) | Resolved (12)
```
where `Issues`/`Questions`/`Suggestions` are the pending ones. Maybe do similar on the dashboard.

This _shouldn't be_ difficult, but I'm always surprised by Codewars so no promises. I'll put this in my todo list.

---

For listing translations by specific users, can someone open a new feature request issue? It doesn't make sense to be embedded in this one.

**Comment by anter69** [comment]
Yes, I check the pending issues on the kata's discourse page. What I suggested is partially resolved by the "translations" button (#1822), although the limited number of users is probably too limiting.

Something similar would be useful for the pending issues/suggestions too. I vaguely remember a manual "hack" of the search url\*, where you could list katas with pending issues, but I can't seem to find it. A built-in button or at least a documented workaround would be nice.

\* something like: https://www.codewars.com/kata/search/?issue=true
